CONTRIBUTORS.txt

```diff
@@ -14,24 +14,27 @@ needs to be listed here.
 - 0xflotus <0xflotus@gmail.com>
 - Adam Nemecek <adamnemecek@gmail.com>
 - Albert Aleksieiev <albert.aleksieiev@gmail.com>
-- Alex Blewitt <alex.blewitt@gmail.com>
+- Alex Blewitt <alblue@apple.com>
+- Alex Reilly <fabiobean2@gmail.com>
 - Andrew Bennett <635596+therealbnut@users.noreply.github.com>
```

Comment:
@therealbnut  if you would like me to add a proper email address here, please reply with the preferred email address.

Reply:
Hi,
andrew@nio.bennnn.net
thanks :)

Reply:
thanks, done!